README.adoc

```diff
@@ -84,14 +89,22 @@ eval $(minikube docker-env)
 
 ==== Build the core riff components
 
-This builds the Function Sidecar, HTTP Gateway, Topic Controller, and Function Controller:
+This builds the Function Sidecar, HTTP Gateway, Topic Controller, Function Controller, and riff CLI:
 
 [source, bash]
 ----
 ./build-function-sidecar
 ./build-http-gateway
 ./build-topic-controller
 ./build-function-controller
+./build-riff-cli
+----
+
+Add the compiled riff CLI to your path or alias it.
+
+[source, bash]
+----
+alias riff="$(go env GOPATH)/src/github.com/projectriff/riff-cli/riff"
```

Comment:
actually for this i was hoping to add a `go install`  step to the cli makefile but this is enough to unblock this PR.